extension/.env.example

```
@@ -0,0 +1,14 @@
+
+# Backend Base URL
+VITE_BACKEND_URL=https://sokuji.kizuna.ai
+
+# Enable Kizuna AI provider (true/false)
+VITE_ENABLE_KIZUNA_AI=true
+
+# PostHog Analytics Configuration
+# Set POSTHOG_KEY to enable analytics tracking
+# Leave empty or unset to disable analytics (recommended for forks)
+POSTHOG_KEY=
+POSTHOG_HOST=https://us.i.posthog.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTHOG_HOST key should go before the POSTHOG_KEY key

(UnorderedKey)